src/__tests__/__snapshots__/documenter.test.ts.snap

```diff
@@ -5103,7 +5114,8 @@ display and produce values.
 Has no effect when \`dateOnly\` is true.
 
 Default: the user's current time offset as provided by the browser.
-",
+
+The property is deprecated, use getTimeOffset instead.",
```

Comment:
```suggestion
This property is deprecated. Use getTimeOffset instead.
```

Reply:
Oh, thanks. I think it was applied to the documenter output and then overwritten. I will update.